packages/trader/src/AppV2/Hooks/useOrderDetails.ts

```diff
@@ -0,0 +1,163 @@
+        : `${tick_count} ${ticks_label}`;
+
+    return {
+        [localize('Reference ID')]: [
+            data.transaction_ids?.buy ? `${data.transaction_ids.buy} (Buy)` : '',
+            data.transaction_ids?.sell ? `${data.transaction_ids.sell} (Sell)` : '',
+        ],
+        [localize('Stake')]: data.buy_price ? `${data.buy_price.toFixed(2)} ${data.currency}` : '',
+        [localize('Duration')]:
+            Number(tick_count) > 0
+                ? ticks_duration_text
+                : `${getDurationTime(data) ?? ''} ${getDurationUnitText(getDurationPeriod(data)) ?? ''}`,
+        [localize('Payout per point')]: data.display_number_of_contracts ? `${data.display_number_of_contracts}` : '',
+    };
+};
+
+// For Multiplier
+const transformMultiplierData = (data: TContractInfo) => {
+    const commonFields = getCommonFields(data);
+    return {
+        [localize('Reference ID')]: commonFields[localize('Reference ID')],
+        [localize('Multiplier')]: data.multiplier ? `x${data.multiplier}` : '',
+        [localize('Stake')]: commonFields[localize('Stake')],
+        [localize('Commission')]: data.commission ? `${data.commission} ${data.currency}` : '',
+        [localize('Take Profit')]: data.limit_order?.take_profit?.order_amount
+            ? `${data.limit_order.take_profit.order_amount.toFixed(2)} ${data.currency}`
+            : localize('Not set'),
+        [localize('Stop loss')]: data.limit_order?.stop_loss?.order_amount
+            ? `${data.limit_order.stop_loss.order_amount.toFixed(2)} ${data.currency}`
+            : localize('Not set'),
+        [localize('Stop out level')]: data.limit_order?.stop_out?.order_amount
+            ? `${data.limit_order.stop_out.order_amount.toFixed(2)} ${data.currency}`
+            : '',
+    };
+};
+
+// For Rise
+const transformRiseData = (data: TContractInfo) => {
+    const commonFields = getCommonFields(data);
+    return {
+        [localize('Reference ID')]: commonFields[localize('Reference ID')],
+        [localize('Duration')]: commonFields[localize('Duration')],
+        [localize('Barrier')]: data.barrier ? data.barrier : '',
+        [localize('Stake')]: commonFields[localize('Stake')],
+    };
+};
+
+// For Turbos
+const transformTurbosData = (data: TContractInfo) => {
+    const commonFields = getCommonFields(data);
+    return {
+        [localize('Reference ID')]: commonFields[localize('Reference ID')],
+        [localize('Duration')]: commonFields[localize('Duration')],
+        [localize('Barrier')]: data.barrier ? data.barrier : '',
+        [localize('Payout per point')]: commonFields[localize('Payout per point')],
+        [localize('Stake')]: commonFields[localize('Stake')],
+        [localize('Take Profit')]: data.limit_order?.take_profit?.order_amount
+            ? `${data.limit_order.take_profit.order_amount.toFixed(2)} ${data.currency}`
+            : localize('Not set'),
+    };
+};
+
+// For Digits
+const transformMatcherData = (data: TContractInfo) => {
```

Comment:
Maybe it will be better to rename `transformDigitData`? Because according to config object below, it's used for all Digits